src/Controls/src/Core/Interactivity/EventTrigger.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Maui.Controls
 	[ContentProperty("Actions")]
 	public sealed class EventTrigger : TriggerBase
 	{
-		static readonly MethodInfo s_handlerinfo = typeof(EventTrigger).GetRuntimeMethods().Single(mi => mi.Name == "OnEventTriggered" && mi.IsPublic == false);
+		static readonly MethodInfo s_handlerinfo = typeof(EventTrigger).GetMethod(nameof(OnEventTriggered), BindingFlags.Instance | BindingFlags.NonPublic, binder: null, types: new Type[] { typeof(object), typeof(EventArgs) }, modifiers: null);
```

Comment:
That looks interesting, I haven't thought of that. I'll give it a try.

Reply:
So, it turns out we cannot do exactly what you suggested because `OnEventTriggered` is an instance method, but I think we don't need the static `s_handlerinfo` at all.